crates/dbs-miniball/docs/CPUVirtualization.md

```diff
@@ -0,0 +1,869 @@
+# CPU 虚拟化
+
+## 概述
+在配置 KVM、guest memory 后，开始配置 vCPU。通过 dbs-boot、dbs-arch、kvm-ioctls crates 完成。
```

Comment:
Pls use English or Chinese&English

Reply:
As I have discussed with @yaoyinnan, he should first finish all the virtualization-related documents in his preferred language. And he could translate it to English when he has extra time. The important part of this project is virtualization knowledge instead of language translation.

But @yaoyinnan, could you add an comment/tip here to tell potential readers why there is only Chinese here?

Reply:
Yes, as @studychao said. Since my current main task is to build a simple virtual machine to sort out the virtualization knowledge points, the document is in Chinese. After I have free time, I will sync it for translation.